app/routes/newsletters/search.js

```diff
@@ -126,8 +143,14 @@ export default class NewsletterInfosSearchRoute extends Route {
     });
   }
 
-  setupController(controller) {
-    super.setupController(...arguments);
+  setupController(controller, model) {
+    super.setupController(controller, model);
+    const params = this.paramsFor('search');
+    controller.searchTextBuffer = params.searchText;
+    controller.mandateesBuffer = params.mandatees;
+    controller.page = params.page;
+    controller.dateFromBuffer = controller.deserializeDate(params.dateFrom);
+    controller.dateToBuffer = controller.deserializeDate(params.dateTo);
     controller.emptySearch = isEmpty(this.paramsFor('search').searchText);
```

Comment:
Define as getter on controller?

Reply:
will check, guessing the entire `setupController` needs to be relooked since `const params = this.paramsFor('search');` needs to go